### DISCUSSION TITLE
Is there a way to enable spell checker on markdown inputs?

### DISCUSSION THREAD

**Discussion by underdpt** [filamentphp/filament#12671] [Help]
### Package

Form builder

### Package Version

v3.2.73

### How can we help you?

I've noticed that there's no spell checking on markdown editor, you can see same wrong text spell checked on textarea input, but not on markdown (left) one:

![imagen](https://github.com/filamentphp/filament/assets/8122137/455748e8-71de-4fbd-9e90-2aabec9832af)

Is there a way to make it work?

**Comment by egidiusmengelberg** [comment]
yeah having the same issue. Currently seems no way to enable it. I could draft a PR to implement this. But since new features for v3 are closed, it seems to not have priority.

**Reply by damms005** [reply]
PRs are accepted for v4 though
